phpstan-baseline.neon

```diff
@@ -1,8 +1,14 @@
 parameters:
     ignoreErrors:
+        - # Disallow VariableMethodCallRule and VariablePropertyFetchRule
+            message: '#^Variable (method call|property access)#'
+            path: .
         - # https://github.com/phpstan/phpstan-phpunit/issues/87
             message: '#^Trying to mock an undefined method [a-zA-Z]*\(\) on class stdClass\.$#'
             path: tests/
+        - # https://github.com/phpstan/phpstan-strict-rules/issues/130
+            message: '#^Call to static method PHPUnit\\Framework\\Assert::.* will always evaluate to true\.$#'
```

Comment:
Cant some of these be removed or they conflict with other errors for psalm?

Reply:
The issue is that we checking our return type.

for instance the method
```
function createDate(): \DateTime {
    return new \DateTime();
}
```
has the following test
```
self::assertInstanceOf(\DateTime::class, createDate());
```

Phpstan is complaining about this being always true. But we might consider it's the purpose of the test to be sure it will be true in the futur.